src/components/PipelineRunDetailsView/PipelineRunDetailsView.tsx

```diff
@@ -116,16 +117,20 @@ export const PipelineRunDetailsView: React.FC<PipelineRunDetailsViewProps> = ({
             key: 'stop',
             label: 'Stop',
             tooltip: 'Let the running tasks complete, then execute finally tasks',
-            isDisabled: !(plrStatus && plrStatus === 'Running'),
-            disabledTooltip: "You don't have access to stop a build",
+            isDisabled: !(plrStatus && plrStatus === 'Running') && canPatchPipeline,
```

Comment:
In succeeded pipeline run's details page actions, if there isn't access to patch pipeline run object the stop and cancel options are enabled.


```suggestion
            isDisabled: !(plrStatus && plrStatus === 'Running') || !canPatchPipeline,
```